google/earthenterprise
# Fixes #765 and #252 - Fix no portables and bad links

Needed to check that portable files actually exist using the full path instead of just the filename.  Should really still add a log message for files that don't exist on Monday.

Tested on #252 and #765 on CentOS 7
Test steps:
1) Create, push, and publish a database using fusion and geserver (to see that links don't break display)
2) Create a symlink in /etc/google/gehttpd/htdocs/cutter/globes
3) Place a portable globe in /etc/google/gehttpd/htdocs/cutter/globes
4) Log in to the server admin page
5) Pushed DB from step 1 is visible
6) Clicking manage portables brings up a window containing the portable globe
7) Add another file into the portable globes file
8) Refresh the portable globes display and ensure the new globe appears
